tests/sentry/api/endpoints/test_organization_dashboards.py

```diff
@@ -219,6 +219,59 @@ def test_get_sortby_mydashboards_and_recently_viewed(self):
             "Dashboard 3",
         ]
 
+    def test_get_sortby_mydashboards_with_owner_name(self):
+        user_1 = self.create_user(username="user_1", name="Cat")
+        self.create_member(organization=self.organization, user=user_1)
+
+        user_2 = self.create_user(username="user_2", name="Pineapple")
+        self.create_member(organization=self.organization, user=user_2)
+
+        user_3 = self.create_user(username="user_3", name="Banana")
+        self.create_member(organization=self.organization, user=user_3)
+
+        user_4 = self.create_user(username="user_4", name="Aapple")
+        self.create_member(organization=self.organization, user=user_4)
+
+        Dashboard.objects.create(title="A", created_by_id=user_1.id, organization=self.organization)
+        Dashboard.objects.create(title="B", created_by_id=user_2.id, organization=self.organization)
+        Dashboard.objects.create(title="C", created_by_id=user_3.id, organization=self.organization)
+        Dashboard.objects.create(title="D", created_by_id=user_4.id, organization=self.organization)
+        Dashboard.objects.create(title="E", created_by_id=user_2.id, organization=self.organization)
+        Dashboard.objects.create(title="F", created_by_id=user_1.id, organization=self.organization)
+
+        self.login_as(user_1)
+        response = self.client.get(self.url, data={"sort": "mydashboards"})
```

Comment:
Does this need to get wrapped with the `with features..` context so it has the table feature flag?